framework.go

```diff
@@ -118,7 +117,7 @@ func (f *Framework) LoadCluster() (*application.Cluster, error) {
 
 	return &application.Cluster{
 		Name:      name,
-		Namespace: name,
+		Namespace: namespace,
```

Comment:
Nice catch 😅

Reply:
Yeah it didn't work otherwise and I was banging my head for an hour trying to figure it out :D